### PR TITLE
Make the chip-tool and darwin-framework-tool logging about pairing clearer

### LINKED ISSUE #19451: [pre-sve] TC-DD-3.1 "PASE Session Established" logs are missing during commissioning
#### Problem
In DD-3.1 step 2
 * expected behavior
       - during PASE session we must:
 > Check on DUT:
CHIP:SC: Received PBKDF param request
CHIP:SC: Peer assigned session ID 18450
CHIP:SC: Found MRP parameters in the message
CHIP:SC: Including MRP parameters in PBKDF param response
CHIP:EM: Piggybacking Ack for MessageCounter:850320430 on exchange: 63762r
CHIP:IN: Prepared unauthenticated message 0xaaaaad340560 to 0x0000000000000000 (0)  of type 0x21 and protocolId (0, 0) on exchange 63762r with MessageCounter:1341084110.
CHIP:IN: Sending unauthenticated msg 0xaaaaad340560 with MessageCounter:1341084110 to 0x0000000000000000 at monotonic time: 0000000000FBA380 msec
CHIP:SC: Sent PBKDF param response

> Check on TH:
Sent PBKDF param request
[1653471961.364996][30157:30162] CHIP:CTL: Setting wifi credentials from parameters
[1653471961.365051][30157:30162] CHIP:CTL: Setting attestation nonce to random value
[1653471961.365166][30157:30162] CHIP:CTL: Setting CSR nonce to random value
[1653471961.365285][30157:30162] CHIP:CTL: Commission called for node ID 0x0000000000000001
[1653471961.365797][30157:30163] CHIP:DL: TRACE: Bluez mainloop starting Thread
[1653471961.366157][30157:30160] CHIP:DL: TRACE: Bus acquired for name C-75cd
[1653471961.382240][30157:30162] CHIP:DL: PlatformBlueZInit init success
[1653471961.393893][30157:30160] CHIP:BLE: BLE removing known devices.
[1653471961.397319][30157:30160] CHIP:BLE: BLE initiating scan.
[1653471961.415415][30157:30160] CHIP:BLE: Device 21:2D:EF:B9:9D:46 does not look like a CHIP device.

 * actual behavior
        - Commissioning is successful but these logs are missing
 
 * system configuration
        -  DUT used : ESP32 
        -  app use: all-clusters-app
        - chip-tool and DUT commit id : 425f1fb916ca154deeb75cd82cd5196778aba152 (pre-sve SHA)
        

 *  logs:
        - esp32: [esp32.txt](https://github.com/project-chip/connectedhomeip/files/8878326/esp32.txt)
        - chip-tool: [th.txt](https://github.com/project-chip/connectedhomeip/files/8878327/th.txt)

### PR DESCRIPTION
**Pull request by bzbarsky-apple**
Make it clear whern PASE vs CASE is done.

Fixes https://github.com/project-chip/connectedhomeip/issues/19451

#### Problem
People being confused about when PASE is done.

#### Change overview
Make the logging clearer.

#### Testing
Looked at logs during commissioning.

### CODE REVIEW

File: examples/chip-tool/commands/pairing/PairingCommand.cpp [hunk 1]
```diff
@@ -144,6 +144,7 @@ void PairingCommand::OnStatusUpdate(DevicePairingDelegate::Status status)
     {
     case DevicePairingDelegate::Status::SecurePairingSuccess:
         ChipLogProgress(chipTool, "Secure Pairing Success");
+        ChipLogProgress(chipTool, "CASE establishment successful");
```
Comment (andy31415) [inline examples/chip-tool/commands/pairing/PairingCommand.cpp:147]: Why not combine the two logs:

`Secure pairing over CASE succes` and
`Secure pairing over PASE success`

Comment (andy31415) [reply examples/chip-tool/commands/pairing/PairingCommand.cpp:147]: I see some tests referencing the string:

src/app/tests/suites/certification/Test_TC_DD_3_5.yaml
src/app/tests/suites/certification/Test_TC_DD_3_7.yaml

however I believe guiding logs by tests (effectively locking in logs) is probably not ideal. If we need to lock in something, we should somehow highlight it as a special string, not make it look like "normal log".

Comment (bzbarsky-apple) [reply examples/chip-tool/commands/pairing/PairingCommand.cpp:147]: I had an explicit request (in https://github.com/project-chip/connectedhomeip/issues/19451) from some of the people writing test plans and scripts to have a log that has "PASE establishment" in it, and to have a separate log with "Pairing success" in it.  I don't know why they want that, but....

Comment (woody-apple) [reply examples/chip-tool/commands/pairing/PairingCommand.cpp:147]: Resolving, given yes, this was a specific request for log lines from testing.

**Review by woody-apple — APPROVED** [review]

**Comment by woody-apple** [comment]
Fast tracking tooling updates.

**Pull request merged**